client/logmon/client.go

```diff
@@ -14,6 +15,8 @@ type logmonClient struct {
 	doneCtx context.Context
 }
 
+const logmonRPCTimeout = 1 * time.Minute
```

Comment:
I set the limit to be too generous, because I was worried about setting it too short.  Happy to follow other's recommendations here.

Reply:
I think this is a fine timeout considering some clients, especially windows, tend to do weird things when they start to become starved of resources.